apps/dialtone-documentation/docs/.vuepress/common/token-utilities.js

```diff
@@ -45,3 +46,138 @@ export function getComposedShadowTokens (theme) {
     });
   return tokens;
 }
+
+// eslint-disable-next-line complexity
```

Comment:
I don't love to ignore the complexity linter rule, but there was no way of categorizing the tokens with a general criteria (they don't always have the category / subcategory name in the same place). So I decided to do a bunch of ifs that will at least be clear to the reader and avoid cumbersome regular expressions.

Reply:
I think it is fine in the case of processing files such as this to ignore complexity rule. The purpose of this function is basically to take a large amount of complexity and structure/simplify it, so it's understandable it's high complexity.